test/readable.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var expect = require('expect');
+var sinon = require('sinon');
```

Comment:
I think I was going to prefer [`jest-mock`](https://www.npmjs.com/package/jest-mock) since it has integrations with `expect`.

What do you think?

Reply:
I'm happy with sinon.